build/migration-guide.md

```diff
@@ -0,0 +1,109 @@
+
+### Step 4:  Deposit Payment and Commit
+
+The final transaction involves committing to the registration and paying the validator's deposit of 100 000 NIM. This transaction can be sent from any address. Despite the address you use to pay the validator deposit, **you must include your validator address** in the transaction’s “public message” field so we can later identify your validator.
+
+To send the transaction manually via wallet, you need the following data:
+
+**Burn Address**: NQ07 0000 0000 0000 0000 0000 0000 0000 0000
+
+**Value**: 100 000 NIM
+
+**Public Message**: Your validator address (e.g. NQ65 SYRU Q915 CMSP G28V 4HL7 C9RP DVAX YLBP)
+
+Alternatively, you can simplify the process by clicking [here](https://wallet.nimiq.com/nimiq:NQ0700000000000000000000000000000000?amount=100000) to access a pre-filled transaction. Enter your validator address as “public message” and proceed with the deposit payment.
+
+💡 Please note that any value below 100 000 NIM will result in permanent loss, and any value above will be burned.
+
+## Validator Activation Tool
+
+The Activation tool facilitates the transition from the PoW chain to the PoS chain. Keep in mind that this tool can only be executed once the pre-stake phase is concluded, specifically during the activation phase.
+
+- Clone and compile the [PoS blockchain](https://github.com/nimiq/core-rs-albatross?tab=readme-ov-file#installation) repository and follow the instructions.
+- Enable the RPC server in the PoW chain. Follow the instructions in [this sample guide](https://github.com/nimiq/core-js/blob/master/clients/nodejs/sample.conf).
+
+### Step 1:  Add your Validator Data into the Configuration File
+
+Once you have your client setup, you can either:
+
+- Go to the file `$HOME/.nimiq/client.toml`, which includes a dedicated validator section at the end, and paste your validator data into this section.
+- Refer to [this guide](becoming-a-validator#configuration).
+
+Ensure that you set your `sync_mode` as `full` or `history` in the consensus part of the configuration file. Only full or history nodes are eligible to be validators.
+
+### Step 2:  Run the Activation Tool
+
+The Activation tool establishes a connection with the PoW chain via RPC, extracting data from your configuration file and crosschecking it with the information in the JSON file within the PoW chain.
+
+To execute the tool successfully, you need to ensure that you are fully synced and in consensus **within the PoW chain**. This requires starting the client via RPC server (this might take a while). You can start it by running the following command:
+
+```shell
+node clients/nodejs/index.js --dumb --network=test --rpc=8648
+```
+
+Once you are in consensus in the PoW chain, proceed to execute the migration tool by running the following command **in the PoS chain**, including the path to the configuration file containing your validator data and specifying the RPC server to be used:
```

Comment:
You must first compile the code, eg. _cargo build --release_ before being able to execute this command